### PR TITLE
Refactor/2489/Migrate add custom config button in ribbonbar

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Migrate 'Add' button for Custom Views in ribbon bar

Ref: #2489, #2318
 
- Migrate 'Add' button to Angular

### CODE REVIEW

File: visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss [hunk 1]
```diff
@@ -1,9 +1,15 @@
 .custom-configs {
```
Comment (shaman-apprentice) [inline visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss:1]: We can remove the empty selector (as suggested by [sonar's detected code smell](https://sonarcloud.io/project/issues?id=maibornwolff-gmbh_codecharta_visualization&pullRequest=2736&resolved=false&types=CODE_SMELL))

File: visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
-	margin-left: 15%;
 }
 
 .custom-configs-component {
-	.custom-configs-button {
+	display: inline-block;
+	width: calc(50% - 2.5px);
```
Comment (shaman-apprentice) [inline visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss:6]: do we need this expensive `calc`? I would prefer alignment with flex or to just hard code some pixel width

**Review by shaman-apprentice — CHANGES_REQUESTED** [review]
I think we should re-add some whitespace to the right. I think giving the two buttons from custom views the same width and whitespaces as the suspicious metrics would be nice?
image

Beside those little things awesome work 👍  This PR will be the first checked checkbox from #2318 not by me 🙂😍

**Comment by shaman-apprentice** [comment]
As the ui is slightly changed, we should probably add an changelog entry.

**Review by Christian-Eberhard — APPROVED** [review]
Looks good to me, only the "white-space" issue remains

**Review by shaman-apprentice — APPROVED** [review]

**Comment by IhsenBouallegue** [comment]
Does this issue have a changelog entry?

**Pull request merged**